CONTRIBUTING.md

```diff
@@ -141,6 +144,57 @@ to email the patch to <gap@gap-system.org>.  You can refer to
 [this tutorial](https://burzalodowa.wordpress.com/2013/10/05/how-to-send-patches-with-git-send-email/)
 on how to do this.
 
+## The reviewing process
+
+Before a change can be incorporated into the code base, our setup requires
+each pull request (PR) to be
+[reviewed](https://help.github.com/articles/about-pull-request-reviews/)
+by at least one GAP developer (i.e. anybody with write access to the
+[gap repository](https://github.com/gap-system/gap/)).
+
+If you don't have write access, you are nevertheless very welcome to submit
+code reviews! Search the
+[pull requests](https://github.com/gap-system/gap/pulls)
+for a PR that you would like to review. Once you have chosen a PR
+(e.g. [PR 2261](https://github.com/gap-system/gap/pull/2261)),
+open the `Files changed` tab
+(e.g. [this](https://github.com/gap-system/gap/pull/2261/files))
+and add your comments or click the green `Review changes` button in the top
+right.
```

Comment:
It's not really an "or" though, is it? In the end, you always must click the green button to submit your review.

Reply:
GitHub also adds a _User reviewed X days ago_ to the timeline if you only add comments. I think this would be equivalent to doing _Comment_ instead of _Approve_ or _Request Changes_.

Reply:
That's not quite accurate, as is the whole description: If you clikc on a line to comment, you can either use the "Add single comment" button -- then a comment is added immediately, but no "User reviewed ... days ago" comment is added. Or you use the "Start a review" button, then added multiple comments; but these stay private until you use the "Review changes" button (and then in there, you can choose from three option, one being to just "comment").

But instead of describing this in detail here, I'd just link to https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/ here. Also, IMHO it's better to avoid excessive use of the `[description](URL)` feature of Markdown and instead write links directly where it makes sense for the text flow -- this file is meant to be readable without a Markdown renderer, too, after all. So how about this:

> To review some code, start by browsing the list of open pull requests (PRs) at
>  <https://github.com/gap-system/gap/pulls> and look for a PR you would like
>  to review. Once you have chosen one, you can comment on its content, and
>  even individual lines changed by it, by following the instructions given on
> <https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/>

Reply:
Ah, I didn't look for a github help page. Awesome!